ee/frontend/mobile-replay/transformers.ts

```diff
@@ -775,35 +770,76 @@ function chooseConverter<T extends wireframe>(
     return converterMapping[converterType]
 }
 
+function convertWireframe(wireframe: wireframe): serializedNodeWithId | null {
+    const children = convertWireframesFor(wireframe.childWireframes)
+    const converter = chooseConverter(wireframe)
+    return converter?.(wireframe, children) || null
+}
+
 function convertWireframesFor(wireframes: wireframe[] | undefined): serializedNodeWithId[] {
     if (!wireframes) {
         return []
     }
 
     return wireframes.reduce((acc, wireframe) => {
-        const children = convertWireframesFor(wireframe.childWireframes)
-        const converter = chooseConverter(wireframe)
-        if (!converter) {
-            console.error(`No converter for wireframe type ${wireframe.type}`)
-            return acc
-        }
-        const convertedEl = converter(wireframe, children)
+        const convertedEl = convertWireframe(wireframe)
         if (convertedEl !== null) {
             acc.push(convertedEl)
         }
         return acc
     }, [] as serializedNodeWithId[])
 }
 
+function isMobileIncrementalSnapshotEvent(x: unknown): x is MobileIncrementalSnapshotEvent {
```

Comment:
Feels like this does a little more than just checking the type on the event
```suggestion
function isValidMobileIncrementalSnapshotEvent(x: unknown): x is MobileIncrementalSnapshotEvent {
```

Reply:
hmmm, I want to split this up more so I'm going to punt this into the future